### PR TITLE
Create libGD_logo-1.svg

### PR DESCRIPTION
**Pull request by rjopek**
Project logo was made libGD_logo-1.svg

### CODE REVIEW

File: docs/libGD_logo-1.svg [hunk 1]
```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<!-- Copyright Rafał Jopek ( rafaljopek at hotmail com ) -->
```
Comment (vapier) [inline docs/libGD_logo-1.svg:3]: i don't think you can claim copyright on an existing image

File: docs/libGD_logo-1.svg [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+<!-- created: 2021-11-20 -->
```
Comment (vapier) [inline docs/libGD_logo-1.svg:4]: these date tags aren't useful.  just delete them.

File: docs/libGD_logo-1.svg [hunk 3]
```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (vapier) [inline docs/libGD_logo-1.svg:1]: why `libGD_logo-1.svg` ?  seems like `logo.svg` would suffice.

File: docs/libGD_logo-1.svg [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    .fil4 {fill:#323B97}
+    .fil5 {fill:#E82C2C}
+    .fil3 {fill:#ED008C}
+    .fil1 {fill:#FFF100}
+    .fil7 {fill:#1C1B17;fill-rule:nonzero}
+    .fil8 {fill:#B2B3B3;fill-rule:nonzero}
+   ]]>
+  </style>
+ </defs>
+ <g id="libGD_logo-1">
```
Comment (vapier) [inline docs/libGD_logo-1.svg:24]: these id attributes aren't needed, so just delete them

File: docs/libGD_logo-1.svg [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+  <metadata id="libGD_logo-1"/>
```
Comment (vapier) [inline docs/libGD_logo-1.svg:25]: this metadata tag isn't used, so delete it

File: docs/libGD_logo-1.svg [hunk 6]
```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<!-- Copyright Rafał Jopek ( rafaljopek at hotmail com ) -->
+<!-- created: 2021-11-20 -->
+<!-- updated: 2021-11-22 -->
+<svg xmlns="http://www.w3.org/2000/svg" xml:space="preserve" width="120mm" height="78mm" version="1.1" style="shape-rendering:geometricPrecision; text-rendering:geometricPrecision; image-rendering:optimizeQuality; fill-rule:evenodd; clip-rule:evenodd"
```
Comment (vapier) [inline docs/libGD_logo-1.svg:6]: i think a bunch of these style values aren't needed, so trim them

File: docs/libGD_logo-1.svg [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+viewBox="0 0 12000 7800"
+ xmlns:xlink="http://www.w3.org/1999/xlink">
+ <defs>
+  <style type="text/css">
+   <![CDATA[
+    .fil6 {fill:#00AA4D}
```
Comment (vapier) [inline docs/libGD_logo-1.svg:12]: should give these fill rules more descriptive names

**Review by vapier — CHANGES_REQUESTED** [review]
you just traced the existing png log from the website right ?

File: docs/libGD_logo-1.svg (hunk 1 above)
Comment (rjopek) [reply docs/libGD_logo-1.svg:3]: Not to the image at all, but to the source code, yes.

File: docs/libGD_logo-1.svg (hunk 3 above)
Comment (rjopek) [reply docs/libGD_logo-1.svg:1]: The very name "logo.svg" says nothing, suggest libGD_logo.svg

File: docs/libGD_logo-1.svg (hunk 1 above)
Comment (rjopek) [reply docs/libGD_logo-1.svg:3]: This is free and unencumbered software released into the public domain.

Anyone is free to copy, modify, publish, use, compile, sell, or
distribute this software, either in source code form or as a compiled
binary, for any purpose, commercial or non-commercial, and by any
means.

**Comment by rjopek** [comment]
I'm sorry but I won't fix these changes.

**Comment by pierrejoye** [comment]
Can we re open this PR? A SVG logo, even if it can improved, would be nice to have :)

**Pull request closed without merging**